src/commit.rs

```diff
@@ -87,3 +87,10 @@ pub fn pre_commit_check(pre_commit_command: Option<String>, message: &str) -> Re
     }
     Ok(())
 }
+
+pub fn git_add_all_modified() -> Result<()> {
+    let output = git_exec(&["add", "-u"])?;
+    std::io::stdout().write_all(&output.stdout)?;
+    std::io::stderr().write_all(&output.stderr)?;
+    Ok(())
```

Comment:
Maybe we would like to check of this have failed and return a error

Reply:
Copying the code from commit would work but then I would have a code duplication, should I factor that out in a new function?

Reply:
You can do something similar to this: https://github.com/alt-art/commit/blob/7e780cab7b7de29563de436ca355da0ea847f6d5/src/commit.rs#L21-L25

Reply:
The commit code do something different.